data/techfusionfm.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "科技聚变",
+  "description": "《科技聚变》（TechFusion）是一档科技主题播客节目，我们谈论有关互联网的一切。您可以访问 TechFusionFM.com 来找到我们，同时您也可以关注我们的 Twitter, Instagram 或微博 @TechFusionFM 。",
+  "author": "Jerry & Dean",
+  "color": "B4B4B4",
+  "logo": "https://techfusionfm.com/images/logo@2x.png",
+  "url": "https://TechFusionFM.com/",
+  "feed": "https://TechFusionFM.com/podcast.xml",
+  "apple": "https://itunes.apple.com/cn/podcast/id1202658654",
```

Comment:
`https://itunes.apple.com/podcast/id1202658654`

Reply:
in accordance with the new URL as https://podcasts.apple.com/podcast/id1202658654